docs/pages/modules/get-started.mdx

```diff
@@ -4,19 +4,115 @@ sidebar_title: Get started
 description: Learn about getting started with Expo modules API.
 ---
 
+import { BookOpen02Icon, Grid01Icon } from '@expo/styleguide-icons';
+
 import { BoxLink } from '~/ui/components/BoxLink';
+import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
-There are two ways to get started with the Expo Module API: you can either initialize a new module from scratch or add the Expo Module API to an existing module. This guide will walk you through creating a new module from scratch, and the [Integrating in an existing library guide](/modules/existing-library) covers the latter case.
+**There are two ways to get started with the Expo Modules API:** you can either initialize a new module from scratch or add the Expo Modules API to an existing module. This guide will walk you through creating a new module from scratch, and the [Integrating in an existing library guide](/modules/existing-library) covers the latter case.
+
+The two recommended flows to create a new module with Expo Modules API:
+
+- Add a new module to an existing Expo application, and use it to test and develop your module.
+- Create a new module in isolation with a generated example project if you want to reuse it in multiple projects or publish it to npm.
+
+Both of these flows are covered in the next sections.
+
+## Adding a new module to an existing application
+
+<Step label="1">
+
+### Creating the local Expo Module
+
+Navigate to your project directory (the one that contains the **package.json** file) and run the following command, which is the recommended way to create a local Expo Module:
+
+<Terminal cmd={[`$ npx create-expo-module --local`]} />
+
+It's best to provide a meaningful module name. However, you can accept the default suggestions for the rest of the questions.
+
+</Step>
+
+<Step label="2">
+
+### Using the module
+
+If you have an **ios** directory in your project that you created using `expo prebuild`, you must reinstall the pods:
```

Comment:
Would be great if `create-expo-module --local` could prompt the user whether he wants us to reinstall pods (one step less in the guide)